diagnostic.md

```diff
@@ -50,21 +46,15 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
-
- <!-- Answer Ends Here -->
+It is relative path, since the path does not have https:// in the beginning.
```

Comment:
This is an absolute path.  The "/" at the beginning of the path represents the root of the system.